packages/terra-form/tests/jest/NumberField.test.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import NumberField from '../../src/NumberField';
+
```

Comment:
Added the example here. https://github.com/cerner/terra-core/pull/410/commits/de1c2e16ab115ac2b74b100d99eb2f85b4904194